examples/diagram-fit/index.html

```diff
@@ -0,0 +1,115 @@
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>BPMN Visualization - Diagram Navigation</title>
+        <link rel="shortcut icon" href="../favicon.ico">
+        <link rel="stylesheet" href="../static/css/spectre.min.css" type="text/css">
+        <link rel="stylesheet" href="../static/css/icons.min.css" type="text/css">
+        <link rel="stylesheet" href="../static/css/main.css" type="text/css">
+        <style>
+            .bpmn-viewport {
+                /* use absolute values for height to ensure that the vertical diagram is not fully displayed when the page is opened. */
+                /* width is relative to the window size */
+                height: 600px;
+                border-style: solid;
+                border-color: #B0B0B0;
+                border-width: 1px;
+                /* This ensures that the parts of the diagram outside of the viewport are not displayed. */
+                overflow: hidden;
+            }
+            #controls {
+                margin-top: 1em;
+            }
+            .menu li {
+                margin-top: 0;
+                /*margin-bottom: 0;*/
+                text-align: right;
+            }
+        </style>
+    </head>
+    <body>
+        <header class="navbar bg-dark p-2">
+            <section class="navbar-section ml-2">
+                <img src="../static/img/logo_64x64.ico" alt="logo" class="logo">
+                <span class="h3 text-light ml-2 lib-name-text">BPMN Visualization Examples</span>
+            </section>
+            <section class="navbar-section">
+                <a href="https://process-analytics.github.io/bpmn-visualization-js/" class="mr-2 btn btn-secondary">Docs</a>
+                <div class="dropdown mr-2">
+                    <a href="#" class="btn btn-primary dropdown-toggle" tabindex="0">
+                        GitHub <span class="icon icon-caret icon-in-the-middle"></span>
+                    </a>
+                    <ul class="menu">
+                        <li><a href="https://github.com/process-analytics/bpmn-visualization-js" class="mr-2 btn btn-link">Library</a></li>
+                        <li><a href="https://github.com/process-analytics/bpmn-visualization-examples" class="mr-2 btn btn-link">Examples</a></li>
+                    </ul>
+                </div>
+            </section>
+        </header>
+        <section class="container" style="margin-top: 3rem">
+            <div class="col-11 col-mx-auto">
+                <h2>Fit on load</h2>
+                <p>If the diagram is too large, it does not fit within the viewport. <br>
+                    Use the radio buttons to choose how fit the diagram in the viewport.
```

Comment:
This is not explicit that the fit is only available on load
It will probably be helpfull to state that to see changes the diagram must be reload after fit type change